docs/walkthrough.md

```diff
@@ -103,33 +112,29 @@ Compare the columns in a new measure file to the study definition:
 * Notice that the columns are parameter values of the new `Measure` instance
 * Notice the additional `value` and `date` columns
 
-Inspect the new deciles chart file:
+Click the new deciles chart file:
 
 * As before, this is generated by the `generate_deciles_charts` action
 * This action is flexible; it will make a deciles chart from each measures file it finds in the *output* directory
 
 ## Requesting a code review
 
-Check that Gitpod has the correct GitHub permissions:
-
-* Visit <https://gitpod.io/integrations>
-* Click *Edit Permissions* under GitHub
-* Click each permission's checkbox and click *Update Permissions*
-* Click *Authorize gitpod-io*
-
-Branch, commit, and publish:
+Branch from `main` and commit:
 
-* Click *Source Control*
-* Create a new branch called `add-cholesterol-measure`
-* Inspect the changes to *analysis/study_definition.py*, *codelists.json*, and *codelists/codelists.txt*
-* Stage the changes 
-* Add "Added cholesterol measure" as a commit message
-* Click *Publish Changes*
+* Click the *Source Control* icon
+* Press <kbd>f1</kbd> and search for "Create Branch"
+* Name the branch e.g. "add-cholesterol-measure"
+* Click *study_definition.py*, *codelists.json*, and *codelists.txt*
+* Inspect the changes to each file
+* Click the plus icon next to each file to stage the changes
+* Add a commit message e.g. "Added cholesterol measure"
+* Press <kbd>Command Enter</kbd> to commit the changes
 
 Create a pull request:
 
-* Click *Create Pull Request*
-* Add a description (use the given title)
+* Press <kbd>f1</kbd> and search for "Crete Pull Request" (The pull request form is broken on Safari.)
```

Comment:
passing through comment/nit (looking as I wondered what you'd changed and if there was anything to feed back into general docs): should be "Create"

Reply:
Lol! Totally, unless there's a special Grecian Pull Request 🤔